test/Staking.test.ts

```diff
@@ -408,4 +417,32 @@ describe('Staking', function () {
       await stakeRegistryPauser.unPause();
     });
   });
+
+  describe('change overlay hex', function () {
+    let sr_staker_1: Contract;
+
+    beforeEach(async function () {
+      await deployments.fixture();
+      token = await ethers.getContract('TestToken', deployer);
+      stakeRegistry = await ethers.getContract('StakeRegistry');
+
+      sr_staker_1 = await ethers.getContract('StakeRegistry', staker_1);
+
+      await mintAndApprove(staker_1, stakeRegistry.address, stakeAmount_1);
```

Comment:
```suggestion
      sr_staker_1 = await ethers.getContract('StakeRegistry', staker_1);

      await mintAndApprove(staker_1, sr_staker_1.address, stakeAmount_1);
```
`stakeRegistry` is not really required